osf_tests/test_external_accounts.py

```diff
@@ -39,3 +42,88 @@ def test_database_is_encrypted(self):
             row = cursor.fetchone()
             for blicky in row:
                 assert jwe.decrypt(bytes(blicky[len(EncryptedTextField.prefix):]), SENSITIVE_DATA_KEY) == self.magic_string
+
+
+class TestEncryptedTextField(TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super(TestEncryptedTextField, cls).setUpClass()
+        cls.field = EncryptedTextField(null=True, blank=True)
+
+
+    def test_ensure_bytes_encodes_no_unicode_in_string_type_str(self):
+        my_value = 'hello'
+        assert_true(isinstance(my_value, bytes))
+        my_str = ensure_bytes(my_value)
+        assert_true(isinstance(my_str, bytes))
```

Comment:
Minor: we aren't using nose asserts in the `osf_tests`; were these tests copied from somewhere else?